.devcontainer/devcontainer.json

```diff
@@ -32,7 +31,5 @@
   },
   "capAdd": [
     "SYS_PTRACE"
-  ],
-  // Uncomment to connect as root instead. More info: https://aka.ms/dev-containers-non-root.
-  "remoteUser": "vscode"
```

Comment:
What's the impact of removing this? I think files will be owned by root which messes with permissions on the host system, not sure anymore.